parrot.rs

```diff
@@ -0,0 +1,73 @@
+use core::sync::atomic::{AtomicUsize, Ordering};
+use kernel::{
+    bindings::msleep,
+    chrdev::Registration,
```

Comment:
If you use `miscdev` instead of `chrdev`, your device will appear automatically in `/dev` once registered.